src/ucs/sys/sys.h

```diff
@@ -342,6 +349,14 @@ void *ucs_sys_realloc(void *old_ptr, size_t old_length, size_t new_length);
  */
 void ucs_sys_free(void *ptr, size_t length);
 
+/**
+ * Get size of CPU cache.
+ *
+ * @param type cache type.
+ */
+ssize_t ucs_get_cache_size(ucs_sys_cache_type_t type);
```

Comment:
The code also has to work with existing configure options:
https://github.com/openucx/ucx/blob/909f05bd5318f5c4c3cf3c7be9e87ce1f3cd1a76/src/ucs/arch/cpu.h#L66

Reply:
```HAVE_CACHE_LINE_SIZE``` is not directly related to cache size and may be configured by user. cache size is runtime configuration

Reply:
yeah, sorry for misleading